tests/scripts/minikube.sh

```diff
@@ -51,11 +51,11 @@ copy_image_to_cluster() {
 }
 
 # configure dind-cluster
-KUBE_VERSION=${KUBE_VERSION:-"v1.7.2"}
+KUBE_VERSION=${KUBE_VERSION:-"v1.7.0"}
```

Comment:
just updated the script. Even though they say v1.7.3 is supported, I cannot start the cluster, v1.7.2 works fine, so updating to v.17.2 for now.